node/src/BaseClient.ts

```diff
@@ -1104,9 +1104,11 @@ export class BaseClient {
      */
     public async getdel(
         key: GlideString,
-        decoder?: Decoder,
+        options?: DecoderOption,
     ): Promise<GlideString | null> {
-        return this.createWritePromise(createGetDel(key), { decoder: decoder });
+        return this.createWritePromise(createGetDel(key), {
```

Comment:
```suggestion
        return this.createWritePromise(createGetDel(key), options);
```

Reply:
this means we wouldn't have to copy data

Reply:
Updated.